### PR TITLE
Liju.jose/105 fix database eslint

### PR DESCRIPTION
**Pull request by LijuJoseJJ**
# Description
fixes eslint error of database microservice.

## Related Issue
fixes#105

## How Has This Been Tested
1. by running integration test
2. by using UI.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+node_modules
+mochawesome-report
+zkp/src/vkIds.json
+zkp/migrations
+zkp/code
+zkp-utils/node_modules
+offchain/migrations
+ui
```
Comment (ronbeilinson) [inline .eslintignore]: new line

File: .prettierignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+package.json
+package-lock.json
```
Comment (ronbeilinson) [inline .prettierignore]: new line

File: .prettierrc.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+
+module.exports = {
+  singleQuote: true,
+  trailingComma: 'all',
+  printWidth: 100,
+  proseWrap: 'always',
+};
```
Comment (ronbeilinson) [inline .prettierrc.js]: new line

Comment (LijuJoseJJ) [reply .prettierrc.js]: done

File: .prettierignore (hunk 2 above)
Comment (LijuJoseJJ) [reply .prettierignore]: done

File: .eslintignore (hunk 1 above)
Comment (LijuJoseJJ) [reply .eslintignore]: done

**Comment by ronbeilinson** [comment]
all tests passing - lgtm :+1:

**Pull request merged**